docs/notebooks/algorithms/bspline_tutorial.ipynb

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
My only comment here is you have a block of commented out code at the bottom of the last cell. Do you think this should be included in the tutorial? If so why? Otherwise, please remove

Reply:
Thank you! It was some old code, removed.